### DISCUSSION TITLE
Keyboard shortcut for Freeze Button?

### DISCUSSION THREAD

**Discussion by neleabels** [dirkwhoffmann/virtualc64#755] [Ideas]
First, many thanks for your work, I greatly enjoy the emulator.

I am using the app mainly for a bit of programming, for which I am using the Action Replay cartridge pretty much. One thing I noticed is that there seems to be no keyboard shortcut to the freezer button - or have I simply overlooked it?

I think it would be a nice addition to the user experience, if you had not to use the mouse to pull down a menu, but simply stayed in the "c64 environment".

Thanks, Nele

P.S. Perhaps a shift-command-R, since control-command-R is the hard reset?

**Comment by bluecursor** [comment]
it would be nice to have a screen to see what all the keys are mapped to or have the ability to map the key ourselves
as like what the DENISE emulator has (Even cartridges buttons)

![Screenshot 2023-05-23 at 8 22 58 PM](https://github.com/dirkwhoffmann/virtualc64/assets/31462938/e95dc2b9-5471-471b-8fd8-edbf417af627)
